README.md

```diff
@@ -84,6 +84,22 @@ From the root dir run:
 sudo env PYTHONPATH=. python <file>
 ```
 
+### Docker installation and usage
+
+#### Pre-requisites
+- [Docker](https://docs.docker.com/engine/install/ubuntu/)
+
+#### installation and usage
+First you need to build the knot-thing-simulator's docker image:
+``` bash
+$ cd <path/to/knot-thing-simulator>
+$ docker build -t knot_thing_simulator .
+```
+Then run the container with the created image:
+``` bash
+$ docker run -d -p 502:502 knot_thing_simulator
+```
```

Comment:
Same here, about skipping lines :grimacing: 

```suggestion
#### installation and usage

First you need to build the knot-thing-simulator's docker image:

``` bash
$ cd <path/to/knot-thing-simulator>
$ docker build -t knot_thing_simulator .
```
Then run the container with the created image:

``` bash
$ docker run -d -p 502:502 knot_thing_simulator
```
```

Reply:
Done